l10n_th_tin_service/views/res_partner_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_partner_form_move_vat" model="ir.ui.view">
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <field name="name_company" position="after">
+                <group>
```

Comment:
Tax ID and Tax Branch display 2 position.
![Selection_153](https://user-images.githubusercontent.com/24691983/108156431-3c9a6200-7113-11eb-9d7d-a1890f6ece99.png)

**My Question**
1. Tax ID and Tax Branch want to move after Company Name ?
2. For individual (not company), Can we fill in only Tax ID ?

Reply:
Answer:
1. Ideally, yes. I want to move "Tax ID" and "Branch" after Company Name only when the radio button "Company" is selected. 
2. When "Individual" is selected, you will not see duplicated "Tax ID" and "Branch". They are at the original position which after the address.